code/game/gamemodes/trifecta/trifecta.dm

```diff
@@ -0,0 +1,115 @@
+#define TOT_COST 5
+#define VAMP_COST 10
+#define CLING_COST 10
+
+
+/datum/game_mode/trifecta
+	name = "Trifecta"
+	config_tag = "trifecta"
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Nanotrasen Representative", "Magistrate", "Chaplain", "Internal Affairs Agent", "Nanotrasen Navy Officer", "Special Operations Officer", "Solar Federation General")
+	restricted_jobs = list("Cyborg")
+	secondary_restricted_jobs = list("AI")
+	required_players = 10
+	required_enemies = 1	// how many of each type are required
+	recommended_enemies = 3
+	secondary_protected_species = list("Machine")
+	var/list/datum/mind/pre_traitors = list()
+	var/list/datum/mind/pre_changelings = list()
+	var/list/datum/mind/pre_vampires = list()
+	var/amount_vamp = 1
+	var/amount_cling = 1
+	var/amount_tot = 1
+
+/datum/game_mode/trifecta/announce()
+	to_chat(world, "<B>The current game mode is - Trifecta</B>")
+	to_chat(world, "<B>Vampires, traitors, and changelings, oh my! Stay safe as these forces work to bring down the station.</B>")
+
+
+/datum/game_mode/trifecta/pre_setup()
+	calculate_quantities()
+	if(GLOB.configuration.gamemode.prevent_mindshield_antags)
+		restricted_jobs += protected_jobs
+	var/list/datum/mind/possible_vampires = get_players_for_role(ROLE_VAMPIRE)
+
+	if(!length(possible_vampires))
+		return FALSE
+
+	if(length(possible_vampires))
```

Comment:
Since you check this just above. Remove and then un-indent the block below.
```suggestion
```

Reply:
done